docs/_documentations/installoncloud.md

```diff
@@ -10,26 +10,146 @@ order: 20
+1. Enter `oc adm policy add-scc-to-group privileged system:serviceaccounts:<che namespace>` to enable privileged containers.
+2. Enter `oc adm policy add-scc-to-group anyuid system:serviceaccounts:<che namespace>` to enable containers to run as root.
+
+## After installing Che
+
+### Adding registries in Che
+After Che is started and running, add the container registry that will be used with Codewind.
+- On OpenShift or other Kube platforms, you can push your images to any Docker registry, such as Dockerhub, Quay.io, Google Cloud Registry (GCR), and more.
+- On IBM Cloud Private, you can push your image to any Docker registry *except* the internal Docker registry
+
+You'll need the following information to complete the instructions to add the registries:
+  - Server: `<registry-to-push-images-to>`
+  - Username: `<Your username>`
+  - Password: `<Your password>`
+
+For information on adding deployment registries to Che, consult our [registry documentation](https://www.eclipse.org/codewind/dockerregistry.html)
+
+## Creating the Codewind workspace
+
+### Confirm the Docker registry secrets.
+Confirm that you added the docker registry secrets in the Che dashboard. Go to **Administration**>**Add Registry** to check for the secrets.
+
+### Creating the Codewind workspace with a Devfile
+The general format for creating a Che workspace via a factory is:
+```
+http://<che ingress domain>/f?url=<hosted devfile URL>
+```
+
+We provide a ready-to-use devfile with the Codewind plug-ins. Enter the following URL to create a workspace from the devfile:
+```
+http://<che ingress domain>/f?url=https://raw.githubusercontent.com/eclipse/codewind-che-plugin/master/devfiles/0.4/devfile.yaml
+```
+
+For other sample devfiles, see https://github.com/kabanero-io/codewind-templates/tree/master/devfiles
+
+### Checking for the Codewind pod
+1. If you are using the Terminal, switch to use the workspace namespace. You can check for the namespace with `kubectl get ns`.
+2. Ensure the projects are cloned into the workspace. You might need to refresh the browser to trigger the clone.
+
+### Configuring Codewind for Tekton pipelines
+From your command line, enter the following commands if you want to use existing Tekton installations with Codewind:
+
+```
+oc apply -f setup/install_che/codewind-tektonrole.yaml
+oc apply -f setup/install_che/codewind-tektonbinding.yaml
+```
+
+For more information about Tekton, see [Getting started with the Tekton Dashboard Webhooks Extension](https://github.com/tektoncd/experimental/blob/master/webhooks-extension/docs/GettingStarted.md).
+
+## After installing Codewind
+
+### Setting the Docker registry
+Upon creating a Codewind workspace. The container registry to deploy your projects to must be set. When you go to create or add an existing project to Codewind, Codewind will prompt you for the registry. See (Docker registry docs) for guidance on using proper container registries
```

Comment:
Maybe, "After creating a Codewind workspace, the container registry..."
Also, there is no link for "Docker registry docs." We should either add the link or remove the sentence I think.